qa/os/src/test/java/org/elasticsearch/packaging/util/Installation.java

```diff
@@ -32,6 +34,8 @@
     public final Path modules;
     public final Path pidDir;
     public final Path envFile;
+    @Nullable
+    private String elasticPassword; // auto-configured password upon installation
```

Comment:
My reasoning is that the auto-configured password becomes a property of the installation, in the same way the configuration is. When the changes to auto-configure for archves are merged, we can migrate all packaging tests to use this password to make requests to the node/cluster